### PR TITLE
Epam meta dev

### PR DESCRIPTION
**Pull request by KateMordasova**

### CODE REVIEW

File: views/core/campaigninsightsdailyagg_rfn.view.lkml [hunk 1]
```diff
@@ -104,16 +85,60 @@ view: +campaigninsightsdailyagg {
...
+  }
+  measure: frequency {
+    type: number
+    value_format_name: "positive_m_or_k"
+    description: "The average number of times each person saw your ad. This metric is estimated."
+    sql: SAFE_DIVIDE( ${sum_of_total_impressions}, ${sum_of_total_reach}) ;;
+  }
+  measure: vtr{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/campaigninsightsdailyagg_rfn.view.lkml:140]: @KateMordasova Could you try using `value_format_name: percent_1` this should let you skip the * 100

File: views/core/campaigninsightsdailyagg_rfn.view.lkml [hunk 2]
```diff
@@ -104,16 +85,60 @@ view: +campaigninsightsdailyagg {
...
+  #A measures with calculations
+  measure: cpm{
+    type: number
+    value_format_name: usd
+    description: "The average cost for 1,000 impressions."
+    sql: SAFE_DIVIDE(${sum_of_total_spend}, ${sum_of_total_impressions} / 1000) ;;
+  }
+  measure: engagement_rate{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/campaigninsightsdailyagg_rfn.view.lkml:110]: @KateMordasova same here as below

File: views/core/campaigninsightsdailyagg__placement_details_rfn.view.lkml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+  measure: link_ctr_placement{
+    type: number
+    value_format:"0.00\%"
+    description: "The number of clicks that your ad receives divided by the number of times your ad is shown."
+    sql: SAFE_DIVIDE( ${sum_of_link_clicks_placement}, ${sum_of_total_impressions_placement}) * 100 ;;
+  }
+  measure: vtr_placement{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/campaigninsightsdailyagg__placement_details_rfn.view.lkml:61]: @KateMordasova  same here

File: views/core/campaigninsightsdailyagg__placement_details_rfn.view.lkml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  #A measures with calculations
+  measure: cpm_placement{
+    type: number
+    value_format_name: usd
+    description: "The average cost for 1,000 impressions."
+    sql: SAFE_DIVIDE(${sum_of_total_spend_placement}, ${sum_of_total_impressions_placement} / 1000) ;;
+  }
+  measure: link_ctr_placement{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/campaigninsightsdailyagg__placement_details_rfn.view.lkml:55]: @KateMordasova Same here

File: views/core/adsetinsightsdailyagg_rfn.view.lkml [hunk 5]
```diff
@@ -1,65 +1,55 @@
...
+    type: number
     value_format_name: usd
-    description: "Pay for a video ad once the user watches a video in its entirety"
-    sql: ${cpcv} ;;
+    description: "The average cost for 1,000 impressions."
+    sql: SAFE_DIVIDE(${sum_of_total_spend_adset}, ${sum_of_total_impressions_adset} / 1000) ;;
+  }
+  measure: link_ctr_adset{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/adsetinsightsdailyagg_rfn.view.lkml:51]: @KateMordasova Same here

File: views/core/adsetinsightsdailyagg__platform_details_rfn.view.lkml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    type: sum
+    value_format_name: "positive_m_or_k"
+    description: "The estimated total amount of money you've spent on your campaign, ad set or ad during its schedule. This metric is estimated."
+    sql: ${total_spend} ;;
+  }
+
+  #A measures with calculations
+  measure: engagement_rate_adset{
+    type: number
+    value_format:"0.00\%"
```
Comment (pthoutam-googler) [inline views/core/adsetinsightsdailyagg__platform_details_rfn.view.lkml:37]: @KateMordasova Same here

File: views/core/campaigninsightsdailyagg_rfn.view.lkml (hunk 1 above)
Comment (KateMordasova) [reply views/core/campaigninsightsdailyagg_rfn.view.lkml:140]: @pthoutam-googler Yes, sure, I will fix it.

**Review by pthoutam-googler — APPROVED** [review]

**Review by thjennifer — COMMENTED** [review]
Please add a space after : 
Should look like `value_format_name: percent_2`

In `adsetinsightsdailyagg__platform_details_rfn.view.lkml` looks like `cpe_adset` should also use `value_format_name: percent_2`. Same for `cpe` in `campaigninsightsdailyagg_rfn.view.lkml`

**Comment by pthoutam-googler** [comment]
@KateMordasova @Krsrinidhi Please look at Jennifer's comments and make changes please.

**Review by thjennifer — APPROVED** [review]

**Pull request merged**